smarts/core/smarts.py

```diff
@@ -224,7 +226,12 @@ def _step(self, agent_actions):
         extras = dict(scores=scores)
 
         # 8. Advance the simulation clock.
-        self._elapsed_sim_time += dt
+        self._elapsed_sim_time = round(self._elapsed_sim_time + dt, 3)
```

Comment:
Good point.  (I added it because the addition of `dt` sometimes caused floating point precision issues, like an `_elapsed_sim_time` equal to, say, `1.9999999` instead of `2.0`.)  I just changed it to never "round too much".